packages/plugin-babel/src/plugin.ts

```diff
@@ -66,6 +68,16 @@ export const getDefaultBabelOptions = (
         require.resolve('@babel/plugin-proposal-decorators'),
         config.source.decorators,
       ],
+      // If you are using @babel/preset-env and legacy decorators, you must ensure the class elements transform is enabled regardless of your targets, because Babel only supports compiling legacy decorators when also compiling class properties:
+      // see https://babeljs.io/docs/babel-plugin-proposal-decorators#legacy
+      ...(isLegacyDecorators
+        ? [
+            [
+              require.resolve('@babel/plugin-transform-class-properties'),
+              { loose: true },
```

Comment:
should `loose` be enabled by default? 

If other plugins set `loose: false`, will it causing conflicts?

Reply:
the `loose` option should be consistent with other preset / plugins

Reply:
Therefore it is not a good idea to fix the value of `loose` in the code